influxdb/templates/pvc.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if and (.Values.persistence.enabled) (not .Values.persistence.useExisting) }}
```

Comment:
is this up to date? it should be using the format->

spec:
  accessModes:
    - {{ .Values.persistence.accessMode | quote }}
  storageClassName: {{ .Values.persistence.storageClass | quote }}
  resources:
    requests:
      storage: {{ .Values.persistence.size | quote }}